apps/stable_diffusion/src/models/model_wrappers.py

```diff
@@ -666,3 +666,44 @@ def controlnet(self):
             return compiled_stencil_adaptor
         except Exception as e:
             sys.exit(e)
+
+	 def __call__(self):
+			# Step 1:
```

Comment:
seems the format is not right

Reply:
yeah it's not right for a few reasons. moving to draft